src/alt-gosling-model/index.ts

```diff
@@ -13,9 +13,13 @@ export function getAlt(
     // get altSpec
     const altSpec = getAltSpec(specProcessed) as AltGoslingSpec;
 
-    // add descriptions
-    treeText(altSpec);
-    dataText(altSpec);
+    try {
+        // add descriptions
+        treeText(altSpec);
+        dataText(altSpec);
+    } catch {
+        console.error('Unexpected error occurred when adding descriptions');
+    }
```

Comment:
When using Editor examples, I was getting uncaught errors around `treeText`. As a workaround, I added this error handling part.